### ISSUE TITLE
Source Code will be Available Soon

### ISSUE THREAD

**Issue by StructML** [StructML/Neural-Network-Prostate#1]
Analysis and Diagnosis of Prostate Cancer Biopsy's Malignancy using Artificial Neural Networks:

      This software program and research will provide an alternative method to prostate cancer diagnosis. Instead of a human pathologists, who is prone to errors, neural networks will be the ones responsible for giving a final decision in cancer diagnosis. Development is currently underway and the code will be open sourced within a few months. There is currently only a prototype code written in Octave. Further work is needed for this to be implemented in a real world scenario.

More Information for Developers:    If you wish to contribute to this project, a few positions are available.
Advanced C++ and C coders are needed to translate the prototyping code into the final software program.
Machine Learning specialists and Data Scientists are needed to improve the accuracy of the diagnosis.
Medical Professionals (Pathologists) are needed to oversee potential issues with this method.
GTK+ developers and computer vision specialists are needed to manipulate images for the Neural Network to analyze.
We also need advanced Octave coders who are willing to prototype the software.

For more information contact www.zandhuang@gmail.com

Researchers and Hospitals are free to use this software. A donation is appreciated.

**Comment by StructML** [comment]
Notice that Octave has officially been switched to Google Tensorflow using the Python interface.
